app/assets/stylesheets/style.css.scss

```diff
@@ -131,14 +179,42 @@ fieldset.tagline {
   font-family: Lato;
 }
 
-#neighborhood h1 {
-  font-family: Lato;
+#neighborhood .one_neighborhood {
+  width: 100%;
 }
 
-#neighborhood_info {
-  padding-top: 4em;
+#neighborhood h1 {
+  font-family: Lato;
 }
 
 #ratings_list {
   padding-top: 3em;
 }
+
+#contributors {
+  border: #B23875 solid 0.2em;
```

Comment:
Color literals like `#B23875` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#B23875` should be written as `#b23875`<br>Properties should be ordered background-image, border, border-radius, height, margin-bottom, margin-top